tests/benchmarks/android_geekbench.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/env python
```

Comment:
@ionela-voinescu do you think this can break our internal testing infrastructure?
Would say no, but I would prefer to get a confirmation from you... ;-)